### PR TITLE
WIP (DO NOT MERGE): persisting language locale

### PR DESCRIPTION
**Pull request by danielweck**
first stab at persisting language locale (Redux Saga boilerplate).

Fixes https://github.com/readium/readium-desktop/issues/435

### CODE REVIEW

**Comment by danielweck** [comment]
@Scarsniik @clebeaupin Could you please help with this one?

Please take a look at the code diff in this PR. As you can see I tried to use the `ConfigRepository` to save the `state.i18n.locale` into persistent storage (alongside the reader settings, etc.).

My understanding of the Redux / Saga boilerplate necessary to make this work is limited (unfortunately my PR does not successfully save the locale in the DB document).

I would appreciate if you could educate me (and the other devs on the team) about the Redux Saga plumbing that is required in Thorium for this simple kind of feature.

For example, this shared knowledge will be useful so that we eventually replace the code that saves the window dimensions / coordinates into a JSON file on the filesystem! (`windowRectangle.json`) See https://github.com/readium/readium-desktop/pull/312

CC @panaC @manuellagrand

File: src/renderer/components/settings/Information.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ interface States {
     placeholder: any;
 }
 
-export class LanguageSettings extends React.Component<Props, States> {
+export class AppInformation extends React.Component<Props, States> {
     private parsedMarkdown: string;
```
Comment (danielweck) [inline src/renderer/components/settings/Information.tsx]: Note that this language-related typo is fixed in this PR, but we could also externalize into a separate PR.

**Comment by danielweck** [comment]
@Scarsniik do you have your own design approach / ideas to solve this problem? Is my PR going in the right direction? Thanks! :)

**Comment by panaC** [comment]
I will see if you want , Cyrille has explained me how the db works..

**Comment by danielweck** [comment]
Great, yes please, I would love your input.
Note that my lack of "expertise" is more about the Redux/Saga flow + state synchronization, rather than about the database per-se. But knowing the DB infrastructure will certainly help! :)
Thanks!

**Comment by danielweck** [comment]
This PR is superseded by https://github.com/readium/readium-desktop/pull/471

**Pull request closed without merging**